spec/config/initializers/rack_attack_spec.rb

```diff
@@ -9,6 +9,7 @@
   let(:cache) { Rails.cache }
 
   before do
+    Rack::Attack.enabled = true
```

Comment:
I confirmed locally that these tests do fail when rack attack is disabled (which is what we'd expect) so I added this to enable them.